src/wwwroot/services/auth.js

```diff
@@ -37,25 +37,41 @@
       isUrlAllowed: isUrlAllowed,
       getDefaultUrl: getDefaultUrl
     };
-
+    var loginSession = null;
     var decodedToken = null;
     var encodedToken = null;
     var temporarilyAuthed = false;
 
-    encodedToken = $window.localStorage.getItem('jwtToken');
-    if (encodedToken) {
-      decodeToken();
-      // verify expiration
-    }
+    init();
     return authService;
 
+    function init() {
+      loginSession = getLoginSession('relayLogin');
+      encodedToken = loginSession? loginSession.jwtToken : null;
+      if (encodedToken) {
+        decodeToken();
+        // verify expiration
+      }
+    }
```

Comment:
You are not keeping backward compatibility with old _jwtToken_ local storage entry, or am I missing something?

Reply:
Yes it is. encodedToken is being loaded from local storage, then I read loginSession from local storage through getLoginSession('relayLogin') and passing the token to encodedToken.
*I will remove getLoginSession to do this more easy to understand and clean.